cmd/zetae2e/run.go

```diff
@@ -41,6 +45,9 @@ For example: zetae2e run deposit:1000 withdraw: --config config.yml`,
 		os.Exit(1)
 	}
 
+	cmd.Flags().String(flagERC20ChainName, "", "chain_name from /zeta-chain/observer/supportedChains")
```

Comment:
`chain_name` is deprecated, we should use `name` instead

Reply:
That field is not live on mainnet yet so we can't use it yet:

```
➜  ~ curl https://zetachain.blockpi.network/lcd/v1/public/zeta-chain/observer/supportedChains
{
  "chains": [
    {
      "chain_name": "eth_mainnet",
      "chain_id": "1"
    },
    {
      "chain_name": "bsc_mainnet",
      "chain_id": "56"
    },
    {
      "chain_name": "btc_mainnet",
      "chain_id": "8332"
    },
    {
      "chain_name": "zeta_mainnet",
      "chain_id": "7000"
    },
    {
      "chain_name": "polygon_mainnet",
      "chain_id": "137"
    }
  ]
}
```